WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1411,6 +1411,10 @@ - (void)savePost
     [self logSavePostStats];
 
     [self.view endEditing:YES];
+
+    //Make sure that we are uploading the latest content on the editor.
+    self.post.postTitle = self.titleText;
+    self.post.content = self.bodyText;
```

Comment:
Also wondering if `autosaveContent` should instead/also be called from the `success` block of `uploadMedia:trackingId:` or from the individual "replace" methods.  Just thinking about any other potential ways that race condition might present itself.